docker/nginx/nginx.conf

```diff
@@ -0,0 +1,26 @@
+upstream graph-node {
+    server graph-node:8000;
+}
+
+map $http_origin $allow_origin {
+    default $http_origin;;
+}
+
+limit_req_zone $binary_remote_addr zone=applimit:10m rate=10r/s;
+limit_req_status 429;
+
+server {
+    listen 8000;
```

Comment:
why are we listening on 8000? and then mapping the port 80 and 443 in docker compose?

This should be 80 right?